vyper/parser/expr.py

```diff
@@ -55,13 +55,14 @@
 
 # var name: (lllnode, type)
 BUILTIN_CONSTANTS = {
-    'EMPTY_BYTES32': (0, 'bytes32'),
-    'ZERO_ADDRESS': (0, 'address'),
-    'MAX_INT128': (SizeLimits.MAXNUM, 'int128'),
-    'MIN_INT128': (SizeLimits.MINNUM, 'int128'),
-    'MAX_DECIMAL': (SizeLimits.MAXDECIMAL, 'decimal'),
-    'MIN_DECIMAL': (SizeLimits.MINDECIMAL, 'decimal'),
-    'MAX_UINT256': (SizeLimits.MAX_UINT256, 'uint256'),
+    'EMPTY_BYTES32': (0, 'bytes32', None),
+    'ZERO_ADDRESS': (0, 'address', None),
+    'MAX_INT128': (SizeLimits.MAXNUM, 'int128', None),
+    'MIN_INT128': (SizeLimits.MINNUM, 'int128', None),
+    'MAX_DECIMAL': (SizeLimits.MAXDECIMAL, 'decimal', None),
+    'MIN_DECIMAL': (SizeLimits.MINDECIMAL, 'decimal', None),
+    'MAX_UINT256': (SizeLimits.MAX_UINT256, 'uint256', None),
+    'ZERO_WEI': (0, 'uint256', {'wei': 1}),
```

Comment:
Do u kno da wei?

Reply:
Actually, can we get this added to the docs (just merged #1613)

Reply:
https://github.com/ethereum/vyper/pull/1623